examples/dodge-the-creeps/rust/src/player.rs

```diff
@@ -58,6 +60,12 @@ impl Area2DVirtual for Player {
     }
 
     fn process(&mut self, delta: f64) {
+        // Don't process if running in editor This part should be removed when
+        // issue is resolved: https://github.com/godot-rust/gdext/issues/70
```

Comment:
Nitpick: missing full stop after "editor"

Reply:
aaaaaaaaaaaaaaaaa that is in fact very understandably infuriating now that i see it